emcc.py

```diff
@@ -1523,6 +1523,7 @@ def default_setting(name, new_default):
         '__emscripten_main_thread_futex',
         '__emscripten_thread_init',
         '_emscripten_current_thread_process_queued_calls',
+        '__emscripten_allow_main_runtime_queued_calls',
```

Comment:
It occurs to me that is kind of sad that these functions that are exported only for internal use probably can't be minimized?   Is that right?   (Or do use the user_exported_functions trick to know that they actually don't need to be fully exported?)   Unrelated to this PR of course.

Reply:
This should be ok. First, the names are minified into `a`, `b` etc. Second, metadce can remove the export if it's not used at all, as it's not a user-requested export.